docs/docs/guides/compatibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: compatibility
+title: Compatibility
+sidebar_label: Compatibility
+---
+
+# Compatibility Table
+
+In order to provide solid, state-of-the arts solutions we often use brand new React Native features,
+that are being introduced all the time. Thats why applications with mismatched Reanimated and React Native
+versions may show unexpected behavior or even fail to build at all. The table below contains official list
+of supported React Native versions.
```

Comment:
```suggestion
```

I'm sorry but this introduction isn't useful and feels a bit forced or chatGPT generated (at least to me!). Generally programmers know what a compatibility table is. A one, simple sentence "The following page contains the compatibility table between Reanimated and React Native versions." is more than enough and not even required tbh